### DISCUSSION TITLE
Date_time column in where between date1 and date2

### DISCUSSION THREAD

**Discussion by signsky2012** [ujjwalguptaofficial/JsStore#365] [Q&A] [answered]
How to select using where query for two dates

**Comment by ujjwalguptaofficial** [comment, marked as answer]
Hi 

You need to pass date object in the column value like this - 

```
select({
    from: "Orders",
    where: {
        orderDate: new Date("1996-07-04T00:00:00.000Z")
    }
});
```

https://ujjwalguptaofficial.github.io/idbstudio/?db=Demo&query=select(%7B%0A%20%20%20%20from%3A%20%22Orders%22%2C%0A%20%20%20%20where%3A%20%7B%0A%20%20%20%20%20%20%20%20orderDate%3A%20new%20Date(%221996-07-04T00%3A00%3A00.000Z%22)%0A%20%20%20%20%7D%0A%7D)%3B

**Comment by ujjwalguptaofficial** [comment]
Added docs for date data type - https://jsstore.net/docs/where/date
